### ISSUE TITLE
insufficient equivalence axiom for 'ciliary basal body'

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-ontology#13316]
The current equivalence axiom for 'ciliary basal body' is this:
<pre>'microtubule organizing center' and ('part of' some 'cell projection')</pre>

This seems insufficiently specific as there are lots of types of cell projections that are not cilia. It seems to me that the correct equivalence axiom would be this:
<pre>'microtubule organizing center' and ('part of' some 'cilium')</pre>

Any objections @ukemi ?

**Comment by ukemi** [comment]
Do any of the other cell projections have microtubule organizing centers? If the only possibility of a 'microtubule organizing center' that is part of a 'cell projection' is a ciliary basal body, then the existing def is correct. It doesn't matter if there are many kinds of cell projections as long as any microtubule organizing center that is part of one is a ciliary basal body. If you think there are microtubule organizing centers that are part of cell projections that are not ciliary basal bodies, then you are correct, the current definition is not sufficient.

**Comment by krchristie** [comment]
I don't know of any other cell projections that have a MOC. But I still don't understand why this is sufficient. If we discovered another type of cell projection later with a MOC, then this wouldn't be sufficient. Why do we not say what we know right now?

**Comment by ukemi** [comment]
I think that logical defs will need to be covered at the next retreat because I am getting lots of questions about them. The short answer is that the more general you can make the genus for the logical definition, the more we can get out of the reasoning. If you really think that we might find a case where the existing def would be made insufficient, then I would change it as you suggest. Otherwise, you are correct, the new MTOCs in structures that are in cell projections that are not cilia would be mis-classified.

**Comment by dosumis** [comment]
@krchristie I agree with you.  The logical def should be tightened.   It's plausible that other projections could have an MOTC + the term is *ciliary* basal body.

**Comment by krchristie** [comment]
@ukemi and I discussed this via skype and he is now in agreement, so I will change this equivalence axiom as I proposed.

**Comment by krchristie** [comment]
I changed the equivalence axiom for 'ciliary basal body' as proposed.

I noticed a couple other things though.

**incorrect assertion for 'microtubule organizing center'**

I also noticed that 'microtubule organizing center' (MOC) is currently asserted to be a SubClass of  'intracellular non-membrane-bounded organelle', but this is not true because a 'ciliary basal body' is part_of a cilium, which is NOT a 'intracellular non-membrane-bounded organelle'. 

I was thinking about changing the asserted SubClass relationship to just say that a microtubule organizing center' is a 'organelle', but I don't really think that a MOC is_a organelle by itself, so I have deleted this assertion. It might be true to say it is 'part_of' an 'organelle', but I'd rather discuss that, so I haven't added anything back for now.

**Incorrect SubClass relationship for 'ciliary basal body'**

Even after making these two relationship changes, I am still seeing that 'ciliary basal body' has a SubClass Of (Anonymous Ancestor) relationship that says "part_of some 'intracellular organelle'. This is inconsistent with the way we decided to place the 'cilium' as an 'organelle' but not as either an intracellular or extracellular organelle.

This seems to be coming from the fact that 'cytoskeleton' is an asserted subclass of 'intracellular non-membrane-bounded organelle' and 'microtubule cytoskeleton' is a subclass of 'cytoskeleton'. @ukemi, maybe check that you agree. Anyway, this seems bigger than I want to deal with on this ticket, and I think we might need to discuss how to proceed.

**Comment by ukemi** [comment]
Hi @krchristie 

Your conclusions above are correct. If we consider the cytoskeleton to be a non-membrane bound organelle, then if something is part of the cytoskeleton it is part of a non-membrane bounded organelle. It seems that whether or not the cytoskeleton is considered an organelle is one of those depends who you ask questions, but it does fit our definition. So I think this the structure is ok. I think you can close this one.

**Comment by krchristie** [comment]
I don't think I understand why this is OK when the relationship I think is incorrect is still present:

<pre>'ciliary basal body'
SubClass Of (Anonymous Ancestor)
- part_of some 'intracellular organelle'</pre>

It seems incorrect to say that the 'ciliary basal body' is part of an 'intracellular organelle' when the organelle that the 'ciliary basal body' is part of, i.e. a 'cilium' is NOT an 'intracellular organelle'.

**Comment by cmungall** [comment]
Unfortunately Protege won't show explanations for anonymous superclass 
expressions, but if you trace up you see it's due to being part of the 
cytoskeleton

**Comment by krchristie** [comment]
Yes, I was aware that it was coming from being part of the cytoskeleton. I missed David's point that the cytoskeleton is itself an organelle. It seems weird that the same structure is simultaneously part of two organelles, the cytoplasm and the cilium, but this is also true for the cytoskeleton within the nucleus, so I'll drop my issue. 

I have already fixed the main reason I opened this ticket, so I'll close it.
